### ISSUE TITLE
Ubuntu 14.04 package required

### ISSUE THREAD

**Issue by eracle** [zatan/ryanair#1]
Ubuntu 14.04 package required
I had to run this before reaching the correct installation of the project:

``` bash
sudo apt-get install  python-dev libffi-dev python-pip libpq-dev libxml2-dev libxslt1-dev
```

**Comment by ikonitas** [comment]
Hi thanks I have updated `README.rst`

**Comment by eracle** [comment]
You Welcome :-D

2015-05-10 13:13 GMT+02:00 Edvinas Jurevicius notifications@github.com:

> Closed #1 https://github.com/zatan/ryanair/issues/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zatan/ryanair/issues/1#event-301126446.

## 

Antonio Ercole De Luca
Linkedin:
http://www.linkedin.com/pub/antonio-ercole-de-luca/1b/340/197
Github:
https://github.com/eracle

To Iterate is Human, recurse Divine.
James O. Coplien.

To be or not to be is true.
G. Boole
